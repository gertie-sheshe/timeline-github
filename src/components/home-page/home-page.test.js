import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HomePage from './home-page';

afterEach(cleanup)

it('Should match Homepage snapshot', () => {
//   expect(true).toBe(true)
  const { asFragment } = render(<HomePage />);
  
  expect(asFragment(<HomePage />)).toMatchSnapshot()
});
