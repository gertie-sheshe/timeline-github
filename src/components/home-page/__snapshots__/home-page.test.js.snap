// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match Homepage snapshot 1`] = `
<DocumentFragment>
  <div
    class="homepage"
  >
    <h1>
      GitHub Timeline
    </h1>
    <p>
      Enter a user's name to see their GitHub timeline :)
    </p>
  </div>
</DocumentFragment>
`;
